Imagine/Filter/Loader/GrayscaleFilterLoader.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Liip\ImagineBundle\Imagine\Filter\Loader;
+
+use Imagine\Filter\Advanced\Grayscale;
+use Imagine\Image\ImageInterface;
+
+/**
+ * GrayscaleFilterLoader - apply grayscale filter
+ *
+ * @author Gregoire Humeau <gregoire.humeau@gmail.com>
+ */
+class GrayscaleFilterLoader implements LoaderInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(ImageInterface $image, array $options = [])
```

Comment:
PR looks good, though you use `[]` which is not supported on php5.3, but we still supports this version, could you changes this to be BC?